WEB_SERVICE/WEB/_HIGH/TLS_FALLBACK_SCSV_NOT_SUPPORTED/recommendation.md

```diff
@@ -0,0 +1,85 @@
+When TLS_FALLBACK_SCSV (Signaling Cipher Suite Value) is not supported, it can lead to potential downgrade attacks. Here are strategies to mitigate this risk:
+
+**Proactive Strategies**
+
+1. **Disable SSL/TLS Version Fallback**: Configure servers to disable fallback to older SSL/TLS versions. This prevents attackers from forcing connections to use vulnerable protocols.
+
+2. **Enforce Minimum TLS Version**: Set a minimum acceptable TLS version (e.g., TLS 1.2) on both client and server sides to prevent downgrade to vulnerable versions.
+
+3. **Regular Security Scans**: Conduct periodic scans of your infrastructure to identify and address any TLS configuration weaknesses.
+
+4. **Monitor for Unusual TLS Negotiation Patterns**: Implement monitoring to detect attempts at forcing protocol downgrades.
+
+**Implementation Examples**
+
+1. Nginx Configuration
+
+To disable SSL/TLS version fallback and enforce a minimum TLS version in Nginx:
+
+```nginx
+server {
+    listen 443 ssl;
+    ssl_protocols TLSv1.2 TLSv1.3;
+    ssl_prefer_server_ciphers on;
+    ssl_ciphers 'ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256';
+    # Other SSL settings...
+}
+```
+
+2. Apache Configuration
+
+For Apache, use the following configuration to achieve similar results:
+
+```apache
+<VirtualHost *:443>
+    SSLEngine on
+    SSLProtocol -all +TLSv1.2 +TLSv1.3
+    SSLCipherSuite ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256
+    SSLHonorCipherOrder on
+    # Other SSL settings...
+</VirtualHost>
+```
+
+3. OpenSSL Command for Testing
+
+Use OpenSSL to test your server's TLS configuration:
+
+```bash
+openssl s_client -connect example.com:443 -tls1_2
+```
+
+This command attempts to establish a TLS 1.2 connection. If successful, it indicates that your server is correctly configured to use modern TLS versions.
+
+**Monitoring for Downgrade Attempts**
+
+### Using fail2ban
+
```

Comment:
![image](https://github.com/user-attachments/assets/9135cefe-18bf-432c-86a0-0684000b9f14)
the second point appears bigger

Reply:
Thanks. It's resolved.